substreams/ethereum-balancer-v3/integration_test.tycho.yaml

```diff
@@ -44,31 +28,14 @@ tests:
         skip_simulation: true # This test is only for creation, simulation tests are below
         creation_tx: "0xaf0f8da4b7b148f156605673c6ac56657f67c47001aadd5b3b64d6cb787daad4"
 
-  - name: test_erc4626_pool_creation # stable pool
-    start_block: 21375196
-    stop_block: 21375216
-    expected_components:
-      - id: "0x89BB794097234E5E930446C0CeC0ea66b35D7570"
-        tokens:
-          - "0x7bc3485026ac48b6cf9baf0a377477fff5703af8"
-          - "0xd4fa2d31b7968e448877f69a96de69f5de8cd23e"
-        static_attributes:
-          manual_updates: "0x01"
-          pool_type: "0x537461626c65506f6f6c466163746f7279"
-          bpt: "0x89bb794097234e5e930446c0cec0ea66b35d7570"
-          fee: "0x2d79883d2000"
-          rate_providers: "0x5b22307865646636336363653462613730636265373430363462373638373838326537316562623065393838222c22307838663465383433396239373033363336343834323163363932646438393766623963306264316439225d"
-        skip_simulation: false
-        creation_tx: "0x536baa8f18a5f9e32b08f695593f47d996a509edc886ccab6f89b001f2d2b4e4"
-
   - name: test_swap_g_usdc # weighted pool
     start_block: 21389576
     stop_block: 21389676
     expected_components:
       - id: "0xf91c11BA4220b7a72E1dc5E92f2b48D3fdF62726"
         tokens:
-          - "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
           - "0x440017A1b021006d556d7fc06A54c32E42Eb745B"
+          - "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
         static_attributes:
```

Comment:
Changing this in the curve PR so that token order doesn't matter, since it was happening for every other test there...